core/src/test/java/io/questdb/test/griffin/ParallelFilterTest.java

```diff
@@ -797,4 +807,47 @@ private void testIn(int jitMode) throws Exception {
                 LOG
         );
     }
+
+    private void testInTimestamp(int jitMode) throws Exception {
+        node1.setProperty(PropertyKey.CAIRO_SQL_JIT_MODE, SqlJitMode.toString(jitMode));
+
+        WorkerPool pool = new WorkerPool((() -> 4));
+        TestUtils.execute(pool, (engine, compiler, sqlExecutionContext) -> {
+                    ddl(compiler, "CREATE TABLE tab (\n" +
+                            "  ts TIMESTAMP," +
+                            "  type INT," +
+                            "  value SYMBOL) timestamp (ts) PARTITION BY DAY;", sqlExecutionContext);
+                    insert(compiler, "insert into tab select (x * 1000 * 1000 * 60)::timestamp, x%10, 't' || (x%10) from long_sequence(10000)", sqlExecutionContext);
+
+                    TestUtils.assertSql(
+                            engine,
+                            sqlExecutionContext,
+                            "select * from tab where ts in '1970-01-01T00:00:00;3m;1d;5'",
```

Comment:
Let's use a more complex filter involving multiple predicates:
```suggestion
                            "select * from tab where ts in '1970-01-01T00:00:00;3m;1d;5' and value = 't3'",
```

Reply:
The designated timestamp filter of this simple form will be processed separately from WHERE clause in `WhereClauseParser`. To test the code you added the column in the filter should be non-designated timestamp

Reply:
Added additional filter and changed the timestamp filter variable to a new one `preciseTs` which is not the designated timestamp